### PR TITLE
Changes for bitrate units.

### PR DESCRIPTION
**Pull request by naushir**
Also a small commit to fix x64 builds with OpenCV.

### CODE REVIEW

**Comment by naushir** [comment]
@davidplowman

File: core/video_options.hpp [hunk 1]
```diff
@@ -13,6 +13,68 @@
...
+
+	void set(const std::string &s)
+	{
+		static const std::map<std::string, uint64_t> match
+		{
+			{ "bps", 1 },
+			{ "b", 1 },
+			{ "kbps", 1000 },
+			{ "k", 1000 },
+			{ "mbps", 1000 * 1000 },
```
Comment (6by9) [inline core/video_options.hpp:29]: Wait for the comment over supporting milli bits/sec.... ;-)

Comment (naushir) [reply core/video_options.hpp:29]: Haha, there was a whole conversation about it this morning :)

File: core/video_options.hpp [hunk 2]
```diff
@@ -13,6 +13,68 @@
...
+			std::size_t end_pos;
+			float f = std::stof(s, &end_pos);
+			bps_ = f;
+
+			for (const auto &m : match)
+			{
+				auto found = s.find(m.first, end_pos);
+				if (found != end_pos || found + m.first.length() != s.length())
+					continue;
+				bps_ *= m.second;
```
Comment (davidplowman) [inline core/video_options.hpp:44]: What happens here if someone wants, for example, "0.5m". Are we going to round off the 0.5 and leave them with zero? Perhaps something like `bps_ = f  * m.second;` would be better?

File: core/video_options.hpp [hunk 3]
```diff
@@ -13,6 +13,68 @@
...
+	void set(const std::string &s)
+	{
+		static const std::map<std::string, uint64_t> match
+		{
+			{ "bps", 1 },
+			{ "b", 1 },
+			{ "kbps", 1000 },
+			{ "k", 1000 },
+			{ "mbps", 1000 * 1000 },
+			{ "m", 1000 * 1000 },
```
[the remark this answers is not in the document]
Comment (naushir) [reply core/video_options.hpp:30]: Will do.

**Pull request merged**